src/FormControls/RadioButton.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export default class RadioButton extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getDOMNode() {
+    return React.findDOMNode(this.refs.input);
+  }
+
+  getValue() {
+    return this.props.value;
+  }
+
+  getChecked() {
+    return this.getDOMNode().checked;
+  }
+
+  renderLabel(input) {
+    if (!this.props.label) {
+      return input;
+    }
+
+    const classes = {
+      'radio': !this.props.inline,
+      'radio-inline': this.props.inline
+    };
+
+    return (
+      <label className={classNames(classes)}>
+        {input}
```

Comment:
Good point! I initially didn't implement it since the wrapping automatically associated the radio button with the label and gave me the green light through the WAVE tool. Having the id would certainly be a good enhancement, though I don't think I'll be making it a required prop.